src/utils/get-functions.test.js

```diff
@@ -76,3 +76,36 @@ test('should mark background functions based on filenames', async (t) => {
     ])
   })
 })
+
+test('should return additional watch dirs when functions requires a file outside function dir', async (t) => {
+  await withSiteBuilder('site-without-functions', async (builder) => {
```

Comment:
[sand] Should we use a different site name, since it does have a function?

Reply:
Yes, good point I'll update it

Reply:
Done in https://github.com/netlify/cli/pull/2143/commits/25be62d9ec0cfdcca080d677c16c5b572041abb1